aten/src/ATen/core/Reduction.h

```diff
@@ -10,6 +10,7 @@ enum Reduction {
   None,             // Do not reduce
   Mean,             // (Possibly weighted) mean of losses
   Sum,              // Sum losses
+  BatchMean,        // Average only over batch dimension
```

Comment:
Is it fine to add this here without it being supported by any of the other losses that accept a reduction? What happens if we pass `"batchmean"` as the reduction to any of the other losses?

Reply:
You are right, that shouldn't be there. It is handled completely on the Python side:

https://github.com/pytorch/pytorch/blob/facff2ec6562d61eab2ccd9f1c16872e54eec92d/torch/nn/functional.py#L2801-L2803

I wonder where the name `"batchmean"` came from if its apparently the same as `"sum"`.